src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -530,6 +534,24 @@ function processItem(item, transaction) {
 }
 ```
 
+#### Retrieving a Transaction
```

Comment:
The same heading appearing multiple times in the same page (3x in this page) will all generate the same anchor link.

Do we have some helper to customize the anchor name? We could use that to make those 3 headings use different anchors.